galaxy_ng/app/api/v1/tasks.py

```diff
@@ -24,6 +29,15 @@
 logger = logging.getLogger(__name__)
 
 
+def parse_version_tag(value):
+    value = str(value)
+    if not value:
+        raise ValueError('Empty version value')
+    if value[0].lower() == 'v':
+        value = value[1:]
+    return semantic_version.Version(value)
```

Comment:
```suggestion
    return semantic_version.Version(value.lstrip('v'))
```

Reply:
@gotmax23 i think this suggestion neglects the case where "v" is an uppercase "V".

Reply:
Ah, I missed the `.lower()`

```suggestion
    return semantic_version.Version("vV")
```

would do that, but it would also remove multiple leading `v`s, which might not be what you want.

/resolve